modules/caddypki/acmeserver/acmeserver.go

```diff
@@ -160,11 +158,68 @@ func (ash Handler) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddyh
 	return next.ServeHTTP(w, r)
 }
 
+func (ash Handler) getDatabaseKey() string {
+	key := ash.CA
+	key = strings.ToLower(key)
+	key = strings.TrimSpace(key)
+	cleaner := regexp.MustCompile(`[^\w.-_]`)
```

Comment:
Could move this into a package-level variable like `var acmeDBKeyCleaner = ...` or something, then we don't have to recompile it each time. Not a big deal though, I don't think this is a hot code path.

Reply:
Ah. That makes sense. I can probably push that quickly, but if you're resolving a merge conflict, I don't want to make it difficult.

Reply:
I haven't started with it, something else came up. If you get to it before I do, that  works too!

Reply:
You want me to rebase and push as well to get rid of the conflict?

Reply:
Sure! We can merge this in then, after the tests finish.

Reply:
I've rebased and can force push to this PR if you'd like, but if you've already fixed the conflict on your end, I'll trash it.

Reply:
Pushed.